jdaviz/components/viewer_data_select.vue

```diff
@@ -168,6 +168,16 @@ module.exports = {
         } else if (this.$props.viewer.reference === 'spectrum-2d-viewer') {
           return (item.ndims === 2 || item.type==='trace') && this.dataItemInViewer(item, returnExtraItems)
         }
+      } else if (this.$props.viewer.config === 'lcviz') {
```

Comment:
Would be cleaner if this can actually live downstream in lcviz but I dunno how to make it happen.

Reply:
Agreed, 100%, but couldn't think of a way to do it short of moving all this logic into python somewhere that downstream can override 😬 .  Hopefully we can revisit in the future and pull out any lcviz-specific logic so that we can be more flexible to other potential future downstream apps